dal/database_connection.py

```diff
@@ -0,0 +1,10 @@
+from beanie import init_beanie
+from models.user import UserModel
+from config import config
+
+async def connect_to_db():
```

Comment:
you dont connect to db you initalize a connection a better name will be
init_db

Reply:
init it in main

Reply:
YOUR BEANIE you dont connect each time you init beanie once through the application.. so it shouldnt be in a model but in the main read about it

Reply:
when i put it in main its ensure that database initialization occurs at a centralized location ?